2021-04-25 15:43:53,533 INFO    MainThread:35622 [wandb_setup.py:_flush():69] setting env: {}
2021-04-25 15:43:53,533 INFO    MainThread:35622 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-25 15:43:53,533 INFO    MainThread:35622 [wandb_init.py:_log_setup():336] Logging user logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_154353-3v3wdwc9/logs/debug.log
2021-04-25 15:43:53,534 INFO    MainThread:35622 [wandb_init.py:_log_setup():337] Logging internal logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_154353-3v3wdwc9/logs/debug-internal.log
2021-04-25 15:43:53,534 INFO    MainThread:35622 [wandb_init.py:init():369] calling init triggers
2021-04-25 15:43:53,534 INFO    MainThread:35622 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-25 15:43:53,535 INFO    MainThread:35622 [wandb_init.py:init():418] starting backend
2021-04-25 15:43:53,535 INFO    MainThread:35622 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-25 15:43:53,554 INFO    MainThread:35622 [backend.py:ensure_launched():123] starting backend process...
2021-04-25 15:43:53,561 INFO    MainThread:35622 [backend.py:ensure_launched():128] started backend process with pid: 35635
2021-04-25 15:43:53,563 INFO    MainThread:35622 [wandb_init.py:init():423] backend started and connected
2021-04-25 15:43:53,565 INFO    MainThread:35622 [wandb_init.py:init():463] updated telemetry
2021-04-25 15:43:53,566 INFO    MainThread:35622 [wandb_init.py:init():482] communicating current version
2021-04-25 15:43:54,263 INFO    MainThread:35622 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-25 15:43:54,264 INFO    MainThread:35622 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-25 15:43:54,460 INFO    MainThread:35622 [wandb_init.py:init():520] starting run threads in backend
2021-04-25 15:43:54,525 INFO    MainThread:35622 [wandb_run.py:_console_start():1524] atexit reg
2021-04-25 15:43:54,526 INFO    MainThread:35622 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-25 15:43:54,527 INFO    MainThread:35622 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-25 15:43:54,530 INFO    MainThread:35622 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-25 15:43:54,530 INFO    MainThread:35622 [wandb_init.py:init():544] run started, returning control to user process
2021-04-25 15:43:58,229 INFO    MainThread:35622 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-04-25 15:43:58,230 INFO    MainThread:35622 [wandb_run.py:_restore():1466] restore
2021-04-25 15:44:00,420 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 15:44:00,704 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 15:44:01,498 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 1951
}

2021-04-25 15:44:01,671 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 1951
}

2021-04-25 15:44:01,773 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1951
  total_bytes: 1951
}

2021-04-25 15:44:01,876 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1951
  total_bytes: 1951
}

2021-04-25 15:44:01,978 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1951
  total_bytes: 1951
}

2021-04-25 15:44:02,081 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1951
  total_bytes: 1951
}

2021-04-25 15:44:02,184 INFO    MainThread:35622 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1951
  total_bytes: 1951
}

2021-04-25 15:44:03,315 INFO    MainThread:35622 [wandb_run.py:_show_files():1838] logging synced files
